src/backend/generator.js

```diff
@@ -308,7 +308,7 @@ exports.createDistDir = function(req, socket, callback) {
     (done) => {
       logger.addLog('Info', 'Creating zip file of the event', socket);
       console.log("==================================Creating zip file");
-      if (emit) socket.emit('live.process', {donePercent:95, status: "Website is being generated"});
```

Comment:
Intially, the percentage values were not in a sorted manner. 95 was coming before 90 as visible in the above code(90 was given for the mailing job which comes later). It was the reason why progress slider was not progressing smoothly.